### ISSUE TITLE
BA.5.2.1 lineage with S:F486I mostly Brazil but also international (124 as of 26.07.22)

### ISSUE THREAD

**Issue by agamedilab** [cov-lineages/pango-designation#752]
Proposal for a sublineage of BA.5.2.1
Earliest sequence: 20.05.2022 (Brazil)
Countries detected: Brazil, Chile

Defining mutations:
T23018A = S:F486I

Couldn't help but notice that tiny Brazilian BA.5 lineage carrying S:F486I.
![grafik](https://user-images.githubusercontent.com/102166254/173244881-cc71f19a-2ea7-4673-8f56-d1c48d7750a2.png)

Only 5 seqs so far:

EPI_ISL_12838776-12838777, EPI_ISL_13164932, EPI_ISL_13206751, EPI_ISL_1322882

**Comment by chrisruis** [comment]
Very small currently so will close for now but can reopen if this grows with an epidemiological event in the future

**Comment by agamedilab** [comment]
Now present in a total of six countries (Brazil, Chile, Mexico, USA, England, Israel). 

![grafik](https://user-images.githubusercontent.com/102166254/175780821-59565460-a31c-4bfa-8cdd-fbd6da90c3c9.png)

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_2ab73_72a4e0.json?branchLabel=aa%20mutations&c=country&label=nuc%20mutations:G23018A

Sequences look clean:

![grafik](https://user-images.githubusercontent.com/102166254/175780872-929d7405-11b7-441f-9970-955d48ae1676.png)

Either dispersal or convergence? Worth monitoring?

@chrisruis

**Comment by agamedilab** [comment]
Quickly expanding cluster of 21 seq. as of 28.6.22. Now present in 8 countries. Worth monitoring? 

Countries detected: Brazil (9), Austria (4), England (2), Israel (2), Chile (1), Mexico (1), New Zealand (1), USA (1)

![1](https://user-images.githubusercontent.com/102166254/176132581-601255ed-556b-4eee-8362-39997f1af17d.PNG)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_27a71_ab4130.json?branchLabel=aa%20mutations&c=gt-S_486&label=nuc%20mutations:C241T,T670G,C1627T,C2790T,C3037T,G4184A,C4321T,C9344T,A9424G,C9534T,C10029T,C10198T,G10447A,C10449A,G12160A,C12880T,C14408T,C15714T,C17410T,A18163G,C19955T,A20055G,C21618T,T22200G,G22578A,C22674T,T22679C,C22686T,A22688G,G22775A,A22786C,G22813T,T22882G,T22917G,G22992A,C22995A,A23013C,T23018G,A23055G,A23063T,T23075C,A23403G,C23525T,T23599G,C23604A,C23854A,G23948T,A24424T,T24469A,C25000T,C25584T,C26060T,C26270T,G26529A,C26577G,G26709A,A27038G,C27807T,C27889T,A28271T,C28311T,A28330G,G28881A,G28882A,G28883C,A29510C

https://cov-spectrum.org/explore/World/AllSamples/Past2M/variants?aaMutations=M%3AD3N%2CS%3A486I&

EPI_ISL_13474619, EPI_ISL_13490361-13490362, EPI_ISL_13490394, EPI_ISL_13228825, EPI_ISL_13350850, EPI_ISL_13350844, EPI_ISL_12838776, EPI_ISL_13460049, EPI_ISL_13410887, EPI_ISL_13460095, EPI_ISL_13371530, EPI_ISL_13206751, EPI_ISL_12838777, EPI_ISL_13428873, EPI_ISL_13425396, EPI_ISL_13164932, EPI_ISL_13437584, EPI_ISL_13490399

**Comment by agamedilab** [comment]
86 seq on usher as of 22.7.22.

![grafik](https://user-images.githubusercontent.com/102166254/180374835-bf5e4572-76dc-4534-bba6-d77dd86d8416.png)

Worth monitoring @InfrPopGen ?

**Comment by agamedilab** [comment]
Growth advantage vs. BA.5 Europe 3 months:

![grafik](https://user-images.githubusercontent.com/102166254/180605930-76902bc6-a9a0-4006-8f97-23337f62adcc.png)
https://cov-spectrum.org/explore/Europe/AllSamples/Past3M/variants?aaMutations=M%3AD3N&aaMutations1=M%3AD3N%2CS%3A486I&analysisMode=CompareToBaseline&

**Comment by FedeGueli** [comment]
@corneliusroemer this sublineage has a mutation in the key RBD mutation (S:486 to I) of BA.4 and BA.5, it is not clear to me why it shows disadvantage in Brazil and wide advantage in Austria and to a lesser extent in France.
i suggest reopening it and monitoring this closely woulde be a good idea .

cc @chrisruis @AngieHinrichs @InfrPopGen

**Comment by FedeGueli** [comment]
Agree with @corneliusroemer , what leves me a little perplexed about this one is that both Austria and France have peculiar sequencing strategy, i am not updated on them but i know for sure Austria is favouring Spike only sequencing and France until some months ago had a genotyping system that probably could influence sequencing then.

Maybe @UlrichElling (https://github.com/UlrichElling) and @Simon-LoriereLab(https://github.com/Simon-LoriereLab) could help in understanding that.

If there is no bias from sequencing strategies the growth of this sublineage would be quite an important thing also thinking at vaccine update ongoing discussions. So designating it to monitor publicly it will be a very good idea

**Comment by agamedilab** [comment]
@FedeGueli Hi, yes Austria favours spike only sequencing [c. 82% (109062 out of 139318 genomes) of all Autrian sequences on GISAID are spike only]. 

The partial sequences are typically uploaded by IMBA lab.

![grafik](https://user-images.githubusercontent.com/102166254/181039309-c06f4119-13e4-4bbf-a95b-5e352ce81653.png)


However, only 8 partial sequences with S:F486I have been uploade so far. 


![grafik](https://user-images.githubusercontent.com/102166254/181040294-2add9892-db06-4ebe-95a9-ec29b6ad23f6.png)


On the other hand there are 32 complete genomes of BA.5.2.1 with S:F486I in Austria.

![grafik](https://user-images.githubusercontent.com/102166254/181040917-e87c0137-912f-4dbb-9671-ed212be5e57b.png)



So in summary, i dont think that the Austrian spike only sequencing skews the picture here too much. Correct me if i am wrong but as far as I know Usher and Covspectrum etc. consider only complete genomes?

**Comment by Simon-LoriereLab** [comment]
Hi Federico, hi all, the national reference lab confirmed that most of the sequencing these days in France is performed via random sampling of positive cases. So the 19 sequences from this linage found in France (they are also dispersed within France) were captured randomly. This is a very small fraction considering the weekly sequencing volume done in France (early signal?), but no bias!
All the best!

**Comment by FedeGueli** [comment]
Thank you very much @Simon-LoriereLab @agamedilab ! Very important informations from both of you!

**Comment by UlrichElling** [comment]
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/109073004/181178061-fac4e09f-6455-464d-a837-fda367780b92.png">
The sequencing strategy in Austria is very custom indeed. We cover the spike protein until aa839 plus a region in ORF1a to distinguish BA4 and BA5 (deletion in BA4) plus the region where the paxlovid resistance is seen plus the region in N that showed positive selection early on. In addition we test for influenza A and have 2 internal human controls. Unfortunately GISAID does not accept too many nnnn stretches and so we can only upload the continuous stretch currently. I tried to contact them several times already, but so far with no success. (any suggestions?) While this custom strategy is a bit odd in some ways it is really cost effective (<20Euros per sample all in since we skip the library prepping stage and add adapters during PCR) and due to the manual assisted annotation we tend to spot curious cases like the BA.1-2 recombinant or the insertions in spike 214 and 248 this way while this is often missed by automatic pipelines.
Sequences by IMP-IMBA are covering all states of Austria in a carefully selected ratio and are systematic random surveillance together with AGES, the official Institute of Health. 
Sequences by LifeBrain (LB, a commercial lab) are usually from Vienna and have been selected by PCR results in the past. They are usually NOT representative. These selective sequencing cases have generated confusion also before.

**Comment by corneliusroemer** [comment]
Off topic for the issue
@UlrichElling have you considered uploading to Genbank/NCBI/ENA in addition to GISAID? They might accept as is, with the Ns.

**Comment by InfrPopGen** [comment]
Thanks for submitting. We've added lineage BF.12 with 59 newly designated sequences, and 4 updated designations from BA.5.2.1. Defining mutation G23018A (S:V486I).
